City-of-Helsinki/employment-services-ui
# THF-472: Add accessibility information to tpr units

added office accessibility information to the TPR units

**Question** 
File - src/components/accordion/AccordionWithIcon.tsx
In lines 32-35, the `groups` array is created, which will later be used to loop the data and set the titles. Should this be moved outside of this AccordionWithIcon if this accordion is used again later?

How to test:
- got to http://localhost:3000/kamppi
- See that end of the page before react and share has accordion item Accessibility information.  Picture below
- Check that the accordion works every language 

- got to page http://localhost:3000/ohjaamo
- you should not see the Accessibility information in this page

<img width="1264" alt="Screenshot 2023-03-23 at 12 04 14" src="https://user-images.githubusercontent.com/42113018/227169643-b48738a6-30d5-4f4c-ba3a-4636304dca22.png">
